### DISCUSSION TITLE
Is it possible to access a related model properties in view

### DISCUSSION THREAD

**Discussion by ossigeno** [livewire/livewire#3423] [1. Help]
In controller using with to include the related entity: 
```
   public $chair = Chair::with("table")->find($id);
   $tables = Table::all();
```

Then in view:
```
 <x-input.select wire:model="chair.table.hashid" id="table_id">
      <option value=""></option>
     @foreach ($tables as $tables)
         <option value="{{ $table->hashid }}">{{  $table->title }}</option>
    @endforeach
</x-input.select>
```

I am trying to bind the select to the hashid of the chair's table instead of the the foreign key table_id from the chair. It is always blank, fails silently.

**Comment by joshhanley** [comment]
@ossigeno you need to make sure you have a `$rules` property if you want to bind to model properties. Have a look here https://laravel-livewire.com/docs/2.x/properties#binding-models

So you would need a rule like
```php
protected $rules = [
    'chair.table.hashid' => 'required', // change required to whatever rule you need
];
```

Hope this helps!
